src/DynamoCoreWpf/ViewModels/Core/ConnectorViewModel.cs

```diff
@@ -188,7 +194,7 @@ public bool PlineVisibility
             }
         }
 
-        public NodeViewModel Nodevm
+        public NodeViewModel StartNode
```

Comment:
Comments?  It's hard to understand what is a start node and end node. So, please write a brief comment.